docs/instructions.adoc

```diff
@@ -78,10 +78,18 @@ If you wish to use {uri-oci-kms}[OCI KMS] to encrypt OKE nodepool boot/block vol
 
 * node_pool_volume_kms_key_id must be provided
 
-If you wish to encrypt data in transit between the instance, the boot volume, and the block volumes.
+If you wish to encrypt data in transit between the instance, the boot volume, and the block volumes in OKE node pool.
 
 * enable_pv_encryption_in_transit must be _true_
 
+If you wish to use {uri-oci-kms}[OCI KMS] to encrypt operator boot/block volume, the following is required:
+
+* operator_volume_kms_id must be provided
+
+If you wish to encrypt data in transit between the instance, the boot volume, and the block volumes in operator.
+
+* enable_operator_pv_encryption_in_transit must be _true_
+
```

Comment:
As above, add subsections so the user know what is being encrypted. Don't rely on the variable name.